govcd/openapi_endpoints_unit_test.go

```diff
@@ -16,12 +18,20 @@ import (
 // * Automatically generated tests where available VCD version does not satisfy it
 // * Automatically generated tests to check if each elevated API version is returned for endpoints that have it defined
 func TestClient_getOpenApiHighestElevatedVersion(t *testing.T) {
+	semverMinVcdApiVersion, err := semver.NewSemver(minVcdApiVersion)
+	if err != nil {
+		t.Fatalf("error parsing 'minVcdApiVersion': %s", err)
+	}
+
 	type testCase struct {
 		name              string
 		supportedVersions SupportedVersions
 		endpoint          string
 		wantVersion       string
 		wantErr           bool
+		// overRideclientMinApiVersion is an option to override default expected version that is
+		// defined in global variable`minVcdApiVersion`
+		overRideclientMinApiVersion string
```

Comment:
Small nit
```suggestion
		// overrideClientMinApiVersion is an option to override default expected version that is
		// defined in global variable`minVcdApiVersion`
		overrideClientMinApiVersion string
```

Reply:
Thanks. Fixed. LOL, that was a failure that made me laugh now :D